### PR TITLE
Gpt integration fix HttpClient

### PR DESCRIPTION
**Pull request by IKonrad98**

### CODE REVIEW

File: News/Migrations/20240923125554_InitialCreate.Designer.cs
Comment (MaxGudinovich) [inline News/Migrations/20240923125554_InitialCreate.Designer.cs:1]: Remove all migrations

File: News/Services/AiIntegrationService.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    private readonly OpenAiOptions _options;
+    private readonly HttpClient _client;
+
+    public AiIntegrationService(IHttpClientFactory httpClientFactory, IOptions<OpenAiOptions> options)
+    {
+        _options = options.Value;
+        _client = httpClientFactory.CreateClient();
+        _client.DefaultRequestHeaders.Add("Authorization", $"Bearer {_options.ApiKey}");
+    }
+
```
Comment (MaxGudinovich) [inline News/Services/AiIntegrationService.cs:21]: I dont remember Avan's comment btw you need to move Http client to HttpRawRequest.

File: News/Migrations/20240923125554_InitialCreate.Designer.cs
Comment (IKonrad98) [reply News/Migrations/20240923125554_InitialCreate.Designer.cs:1]: they have been removed here

File: News/Services/AiIntegrationService.cs (hunk 1 above)
Comment (IKonrad98) [reply News/Services/AiIntegrationService.cs:21]: you can check the comments. They are there. 
I have implemented changes based on those comments. 
If something more is needed, I will ask you to continue writing, as I won’t have time for the next couple of days due to work

**Pull request merged**